.vscode/tasks.json

```diff
@@ -1,31 +0,0 @@
-{
```

Comment:
@caetanator why have you removed this?
(Unfortunately there are no vscode users in the ImageSharp core team, but some of our users find this very useful.)

Reply:
Hi,

I some times (every VS2016 update), have to delete all the configurations of VS2016 in order to make it work again.

In one of this "cleanups", mai have not been add to my version.

I don't think that .vscode interfer with the VS2016 configuations, but was unable to make it load this project. The problem end up to be a bug on GhostDoc plug-in.